cumulus/zombienet/zombienet-sdk/tests/sync_blocks/sync_blocks_from_tip_without_connected_collator.rs

```diff
@@ -0,0 +1,112 @@
+// Copyright (C) Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+use anyhow::anyhow;
+
+use cumulus_zombienet_sdk_helpers::assert_para_throughput;
+use polkadot_primitives::Id as ParaId;
+use subxt::{OnlineClient, PolkadotConfig};
+use zombienet_sdk::{LocalFileSystem, Network, NetworkConfigBuilder};
+
+const PARA_ID: u32 = 2000;
+const BEST_BLOCK_METRIC: &str = "block_height{status=\"best\"}";
+
+#[tokio::test(flavor = "multi_thread")]
+async fn sync_blocks_from_tip_without_connected_collator() -> Result<(), anyhow::Error> {
+	let network = initialize_network().await?;
+
+	let relay_alice = network.get_node("alice")?;
+
+	let relay_client: OnlineClient<PolkadotConfig> = relay_alice.wait_client().await?;
+
+	log::info!("Ensuring parachain making progress");
+	assert_para_throughput(
+		&relay_client,
+		10,
+		[(ParaId::from(PARA_ID), 9..11)].into_iter().collect(),
+	)
+	.await?;
+
+	let para_ferdie = network.get_node("ferdie")?;
+	let para_eve = network.get_node("eve")?;
+
+	log::info!("Ensuring ferdie and eve are syncing");
+	assert!(para_ferdie
```

Comment:
One thing that is of interest, we could add a check here to verify that we only have the expected number of peers. Since we are using `--reserved-only` we could be paranoid and check that we only have the X peers connected there.

Reply:
Done :)